tests/test_builders/test_build_gettext.py

```diff
@@ -96,7 +100,7 @@ def test_msgfmt(app):
                 'msgfmt',
                 'en_US.po',
                 '-o',
-                Path('en', 'LC_MESSAGES', 'test_root.mo'),
+                str(Path('en', 'LC_MESSAGES', 'test_root.mo')),
```

Comment:
`subprocess.run` can take `Path`s. However, the `args` variable was previously defined without a type hint and so `mypy` understands it to be `list[str]`. We can either do the casting here or set `args: list[str | Path] = [...]` earlier. I chose this but I'm not concerned either way.